css/style.css

```
@@ -16,4 +15,98 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav {
+  padding: 20px 40px 3px 60px;
+  background-color: rgb(247, 244, 241);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+.nav_link > img {
+  width: 50px;
+  height: 40px;
+}
+
+.nav_link ul,
+li {
+  display: flex;
+  font-size: 0.8rem;
+  padding-left: 15px;
+}
+li:active {
+  color: red;
+}
+.main-container {
+  background-image: url(/img/first-background.jpg);
+  height: 350px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 0.8rem;
+  line-height: 15px;
+}
+.main-container>button{
+
+    background-color: red;
+    border: none;
+  padding:5px 10px;
+  font-size: 0.6rem;
+}
+
+.container-services{
+    padding-top: 35px;
+    text-align: center;
+}
+.services>img{
+    width: 40px;
+    height: 40px;
+}
+.container{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+
+}
+.services{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-size: 0.8rem;
+}
+hr{
+    width: 80%;
+}
+
+.social-icon>a>img{
+    width: 10px;
+    height: 10px;
+    border: 0.5px solid grey; 
+    border-radius: 50%;
+    padding:5px;
+
+}
+.footer{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;  
+    align-items: center;
```

```suggestion
    text-align: center;
```
We can use `text-align: center` to centre the text and links instead of using flexbox